sonaranalyzer-dotnet/src/SonarAnalyzer.Utilities/Rules.Description/S3366.html

```diff
@@ -22,6 +22,9 @@ <h2>Noncompliant Code Example</h2>
   }
 }
 </pre>
+<h2>Exceptions</h2>
```

Comment:
Is this implemented?

Reply:
If you read the updated tests you will see that I have checked this is the case.